.github/workflows/e2e.yaml

```diff
@@ -44,6 +44,8 @@ jobs:
           ./mvnw --batch-mode -f test/e2e/pom.xml -pl e2e-base clean install
       - name: Single Node Tests(JDK8)
         run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-single-service
+      - name: Single Http API with Ningx Lua Tests(JDK8)
+        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-http-api-with-nginx-lua
```

Comment:
Move this to `Compatibilities` section. This is HTTP + LUA compatible test. And we could run test faster there.